### PR TITLE
Scalafix provides

### PR DESCRIPTION
**Pull request by swoogles**

### CODE REVIEW

File: scalafix/rules/src/main/scala/fix/Zio2Upgrade.scala
Comment (swoogles) [inline scalafix/rules/src/main/scala/fix/Zio2Upgrade.scala:696]: This comment explains the core point of this PR.
Let me know if there are other `provide` variants that we should try to automate!

**Review by adamgfraser — APPROVED** [review]

**Comment by adamgfraser** [comment]
@swoogles May be worth waiting until #6257 gets merged and incorporating that.

**Comment by swoogles** [comment]
> @swoogles May be worth waiting until #6257 gets merged and incorporating that.

Ah yeah, looks like that will simplify this and turn it into another simple rename operation. 

So much for the clever bits I used here! 😛

**Comment by adamgfraser** [comment]
@swoogles That PR is merged if you want to update.

**Pull request merged**
